### PR TITLE
Fixed some docs error and upgraded netty-all

### PR DESCRIPTION
**Pull request by 1Teng**
1.fixed some pages not found errors in Markdown files and some typo errors
2.upgraded netty-all to 4.1.86.Final

### CODE REVIEW

**Comment by aiceflower** [comment]
LGTM

File: pom.xml [hunk 1]
```diff
@@ -364,6 +364,14 @@
         <type>pom</type>
         <scope>import</scope>
       </dependency>
+      <dependency>
```
Comment (casionone) [inline pom.xml:367]: netty-bom  has been seted in parent pom.xml 
and using in linkis-commons/linkis-module/pom.xml with 
```
    <dependency>
      <groupId>io.netty</groupId>
      <artifactId>netty-all</artifactId>
    </dependency>
```
There is no need to add this configuration

**Comment by casionone** [comment]
Third-party Dependencies Check failed 
`tool/dependencies/known-dependencies.txt`  the  `netty-3.6.2.Final.jar`  record can remove  
and netty info need to update

**Review by jackxu2011 — DISMISSED** [review]
LGTM

**Comment by jackxu2011** [comment]
Now the dev branch is 1.3.2, and something has been changed.  Can you push a new pr to dev-1.3.2?

**Review by casionone — APPROVED** [review]

**Review by peacewong — APPROVED** [review]
LGTM.

**Pull request merged**